components/cart.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import style from "../styles/Cart.module.css";
+
+const Cart = () => {
+
+const addToCart = () => {
+ 
+}
+
+const qtyDropdown = () => {
+
+}
+
+const buyNow = () => {
+
+}
+const addToList = () => {
+
+}
+
+    return (
+    <div className={`${style.container}`}> 
+        <div>
+            <div className={`${style.pricing}`}>
+                <div className="numbers">
+                    <span id="symbol">$</span>
+                    <span id="dollars">17</span>
+                    <span id="decimal">.</span>
```

Comment:
Possible unnecessary spans, why have each letter on its own span?

Reply:
was going with the possibility of dynamic adjusting to the price

Reply:
oh no thats the old ones, I changed those in the latest push, thats how I got the cents to be small and to the top